multihost_test/bz_automation/test_bz_automation.py

```diff
@@ -0,0 +1,41 @@
+
+import pytest
+import subprocess
+import os
+import time
+
+
+def exceute_cmd(multihost, command):
+    cmd = multihost.client[0].run_command(command)
+    return cmd
+
+
+@pytest.mark.tier1
+class TestPamBz(object):
+    def test_segmentation(self, multihost):
+        """
+        :title: Groupdel gives segmentation fault when using '-P'
+        :id: f86daef6-540c-11ec-a7c2-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1986782
+                   https://bugzilla.redhat.com/show_bug.cgi?id=2024834
+        """
+        for i in ["mkdir -p $HOME/custom_root/etc/",
+                  "touch $HOME/custom_root/etc/group",
+                  "groupadd -P $HOME/custom_root test_group",
+                  "groupdel -P $HOME/custom_root test_group"]:
```

Comment:
I think that you should check the return value for this command.

Reply:
the test will fail if this command fails . So no need to check the output separately

Reply:
Ok. Makes sense.